deploy/helm/sumologic/templates/otelcol-deployment.yaml

```diff
@@ -39,6 +39,8 @@ spec:
         env:
         - name: GOGC
           value: "80"
+{{- $ctx := .Values -}}
+{{ include "fluentd.envs" (dict "Context" $ctx "Type" "traces")}}
```

Comment:
Why do we call it `flutend.envs` if we use it for otelcol as well?

Reply:
because we didn't use it for otelcol before 😅